src/main/resources/docker/dockercompose/docker-compose-pg.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:14.1-alpine
```

Comment:
this is an idea to have all services in one docker-compose file, finally I will just put in command line docker-compose up and all should start

Reply:
later we can also add PolandSalaryCalculator here, so end user just clone your repository, will run docker-compose up and PolandSalaryCalculator shoudl work together with db

Reply:
first done

Reply:
all done